### PR TITLE
Add tinylog as logging framework

### PR DESCRIPTION
**Pull request by pmwmedia**

### CODE REVIEW

**Comment by lauris** [comment]
Please change the link to github repo of the project.

File: README.md [hunk 1]
```diff
@@ -411,6 +411,7 @@ Projects with over 500 stargazers are in bold.
 * **[Shapeless ★ 2002 ⧗ 0](https://github.com/milessabin/shapeless)** - A type class and dependent type based generic programming library for Scala.
 * [Simulacrum ★ 484 ⧗ 2](https://github.com/mpilquist/simulacrum) - First class syntax support for type classes in Scala.
 * [Stateless Future ★ 165 ⧗ 35](https://github.com/qifun/stateless-future) - Asynchronous programming in fully featured Scala syntax.
+* [tinylog](https://tinylog.org/v2/) - Lightware logging framework with native logging API for Scala.
```
Comment (tkroman) [inline README.md]: lightware -> lightweight?

**Comment by pmwmedia** [comment]
Both fixed!

**Comment by longcao** [comment]
Thanks!

**Pull request merged**